services/client-api/index.js

```diff
@@ -1,67 +1,44 @@
 const express = require('express');
 const handler = require('../../infrastructure/db/handler.js');
+const responseBuilder = require('./responseBuilder');
 const { initDb } = require('../../infrastructure/db/');
 const logger = require('../../infrastructure/logging/logger.js')
 const auth = require('../auth-api/verifyUserMiddleware');
 const app = express()
 const port = 5867;
 initDb();
 app.use(express.json());
+
 app.post('/login', [auth.verifyLogin,
     auth.login]);
 
-app.get('/home', [auth.validateToken], (req, res) => {
-    res.status(200).send({auth : true, id : req.userId, emailAddress: req.emailAddress});
-});
-
-app.route('/boards')
-    .get((req,res) => {
-        let id = req.params.id || 1;
-        handler.getBoards(id).then((boards) => {
-            res.send(boards);
-            logger.logger.info("GET request");
-        });
-    })
-    .post((req, res) => {
-        handler.addBoard({ id : req.body.id, label : req.body.label, folderId: req.body.folderId }).then(() => {
-            res.send("Added a new board");
-            logger.info("Added a new board");
-        });
-    })
-
-app.get('/boards/:boardId', (req, res) => {
-    handler.getBoard({ boardId : req.params.boardId }).then((board) => {
-        res.send(board);
-    });
-});
-
-app.post('/boards/:groupId/task', (req, res) => {
-    handler.addTask({ label : req.body.label, metadata : req.body.metadata, teammemberId : req.body.teammemberId, groupId : req.params.groupId });
-    logger.info("Added a new task");
-});
-
-app.get('/boards/:boardId/teamMembers', (req, res) => {
-    handler.getBoardTeammembers(req.params.boardId).then(teammembers => {
-        res.send(teammembers);
-    });
-});
+app.get('/folders', [auth.validateToken, responseBuilder.getFoldersJson]);
+app.get('/teamMembers/:teamMemberEmail',[ auth.validateToken, responseBuilder.getTeammemberJson]);
+app.get('/boards', [auth.validateToken, responseBuilder.getBoardsJson]);
+app.get('/boards/:boardId', [auth.validateToken, responseBuilder.getBoardJson]);
+app.get('/boards/:boardId/teammembers', [auth.validateToken, responseBuilder.getBoardsTeammembersJson]);
 
-app.post('/folders', (req,res) => {
-    handler.addFolder({ id : req.body.id, label : req.body.label });
+app.post('/folders',[auth.validateToken], (req,res) => {
+    handler.addFolder({label : req.body.label, teammemberId : req.teammemberId});
     res.send("Added a new folder");
     logger.info("Added a new folder");
 });
+app.post('/boards', [auth.validateToken],(req, res) => {
+    handler.addBoard({ teammemberId : req.teammemberId, label : req.body.label, folderId: req.body.folderId }).then(() => {
+        res.send("Added a new board");
+        logger.info("Added a new board");
+    });
+});
 
-app.post('/groups', (req, res) => {
+app.post('/groups',[auth.validateToken], (req, res) => {
```

Comment:
why for some endpoints you are actually declaring their behaviour here instead of the at handler level as the other endpoints?